pageserver/src/tui.rs

```diff
@@ -171,13 +171,8 @@ pub fn ui_main() -> Result<(), Box<dyn Error>> {
         })?;
 
         // If ther user presses 'q', quit.
-        if let Event::Input(key) = events.next()? {
-            match key {
-                Key::Char('q') => {
-                    break;
-                }
-                _ => (),
-            }
+        if let Event::Input(Key::Char('q')) = events.next()? {
```

Comment:
The 'match' syntax makes sense again as soon as we add another key to the UI. Of course, we can easily change this back when that happens.

Reply:
I think you're right and it makes sense to leave this alone.